blockers.yaml.example

```diff
@@ -60,3 +63,13 @@ job7:
         - 852
     jira:
         - 0
+
+# You can have build number for blockers included in report
```

Comment:
```suggestion
# You can have build numbers for blockers included in report
```
Could you may go into a little more detail as to what exactly this does and why users should use it?